cucumber-core/src/test/java/io/cucumber/core/options/CommandlineOptionsParserTest.java

```diff
@@ -84,7 +95,7 @@ void has_version_from_properties_file() {
     }
 
     private String output() {
-        return new String(out.toByteArray(), StandardCharsets.UTF_8);
+        return out.toString(StandardCharsets.UTF_8);
```

Comment:
I don't think this works on 8. I would be okay with using a different test source and target level though.

https://maven.apache.org/plugins/maven-compiler-plugin/testCompile-mojo.html#testSource

Reply:
True, this is Java 10, so I reverted.